src/main/java/org/broadinstitute/hellbender/tools/spark/sv/FindBreakpointEvidenceSpark.java

```diff
@@ -1314,13 +1261,10 @@ public KmerAndCount read(final Kryo kryo, final Input input, final Class<KmerAnd
      * It knows which breakpoint(s) a read belongs to (if any) by kmerizing the read, and looking up each SVKmer in
      * a multi-map of SVKmers onto intervalId.
      */
```

Comment:
In the comment, doesn't this really go from a stream of reads to a stream of `<intervalId, SVKmer>` pairs?